packages/mutation-testing-elements/src/style/globals.scss

```diff
@@ -0,0 +1,3 @@
+// Global variables loaded in mutation-test-report-app
+
+$mte-drawer-height-half-open: 120px; // Leave room for 2 lines of text
```

Comment:
Shouldn't this be a CSS variable in theme.css?

Reply:
No, it is a hardcode variable used in a couple of sass files. I didn't want to pollute CSS variables where we're not switching the values per theme. I thought to a group such settings in `globals.scss`. I'm open to suggestions 🤷‍♀️.